en/security.rst

```diff
@@ -18,8 +18,298 @@ ACL (Access Control List)
+
+By default, the key file is created with the name of **<database-name>_keys** at the location where the data volume is created when creating a database using **cubrid createdb** utility. Without additional configuration for the key file, this key file is automatically used. The location of the key file te be used can be changed by a system parameter. For more information, see :ref:`disk-parameters`.
+
+The key file can contain several master keys (up to 128). A master key among those keys is set on the database to encrypt the database, data keys technically. One master key is created and set by default when the key file is created, and DBA can add, delete, change, and search keys using the TDE utility (:ref:`TDE utility<tde-utility>`). When deleting a key, the key to delete must exist in the key file, and the key set on the database currently cannot be removed. When changing a key to set to encrypt database, both the previously key set on the database and the key to be set must exist in the key file. Through key inquiry, you can check the number of keys and creation time of them, and you can check the current key set on the database and setting time.
+
+.. code-block:: bash
+
+	$ cubrid tde --show-keys testdb
+	Key File: /home/usr/CUBRID/databases/testdb/testdb_keys
+
+	The current key set on testdb:
+	Key Index: 2
+	Created on Fri Nov 27 11:14:54 2020
+	Set     on Fri Nov 27 11:15:30 2020
+
+	Keys Information: 
+	Key Index: 0 created on Fri Nov 27 11:11:27 2020
+	Key Index: 1 created on Fri Nov 27 11:14:47 2020
+	Key Index: 2 created on Fri Nov 27 11:14:54 2020
+	Key Index: 3 created on Fri Nov 27 11:14:55 2020
+
+	The number of keys: 4
+
+.. note:: **Creating a database using an existing key file**
+
+    If you want multiple databases to share an existing or new key file, you can create a database after setting the path of the key file by using the **tde_keys_file_path** system parameter. For more information on the system parameters, see :ref:`other-parameters`.
+
+.. _tde-enc-target:
+
+Encryption Target
+-----------------
+
+.. _tde-enc-perm:
+
+Permanent Data Encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The encrypted table data and all index data created on the table are encrypted. For more information on the encrypted table, see :ref:`create-tde-table`.
+
+.. _tde-enc-temp:
+
+Temporary Data Encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to persistent data such as tables, temporary data created during queries related to encrypted tables are also encrypted. For example, all temporary data created in the process of executing a query such as `SELECT * FROM tde_tbl ORDER BY att1` or creating an index on `tde_tbl` are encrypted when it is written to disk. For more information on temporary data, see :ref:`temporary-volumes`.
```

Comment:
```suggestion
In addition to persistent data such as tables, temporary data created during queries related to encrypted tables are also encrypted. For example, all temporary data created in executing a query such as `SELECT * FROM tde_tbl ORDER BY att1` or creating an index on `tde_tbl` are encrypted when it is written to disk. For more information on temporary data, see :ref:`temporary-volumes`.
```

- comment: here ' the process of' might be unnecessary, consider removing it :).